platforms/documentation/docs/src/snippets/plugins/dependenciesBlock/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,13 @@
+plugins {
```

Comment:
🤔 As we have root `build.gradle` files for these, is there a need for the `settings.gradle` files?

Reply:
All builds need to have a settings file to denote the root.  Maybe the way we run these snippets, we might force this to be the root directory, but I didn't try that.

Reply:
I don't think this is necessary for snippets.

Reply:
I didn't check exhaustively, but every snippet I checked had a settings file: https://github.com/gradle/gradle/tree/master/platforms/documentation/docs/src/snippets/developingPlugins